### PR TITLE
update pin_subpackage

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @isuruf and @CJ-Wright.

PS: I'll remove build number `0` once this one is merged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,12 +11,12 @@ source:
     - makefile.patch
 
 build:
-  number: 0
+  number: 1
   skip: True  # [win]
   run_exports:
     # SONAME changes at minor revs.
```
Comment (isuruf) [inline recipe/meta.yaml]:
```suggestion
     # SONAME changes at patch revs.
```

**Comment by isuruf** [comment]
Does anyone remember the reason for this change? https://abi-laboratory.pro/?view=timeline&l=giflib looks okay except for the renaming of one private symbol `reallocarray` to avoid clashes with glibc.

**Comment by isuruf** [comment]
There was some discussion at https://gitter.im/conda-forge/conda-forge.github.io?at=5c8fa1dd9d9cc8114ade7b97 around the same time, but no reason there either,

**Comment by ocefpaf** [comment]
Sorry. We lost track of it. Do you want to relax the pin?

**Pull request merged**